### PR TITLE
Preparing for 3.0.1 release

### PR DESCRIPTION
**Pull request by singhravidutt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ To add a dependency on Cloud Storage connector using Maven, use the following:
 <dependency>
   <groupId>com.google.cloud.bigdataoss</groupId>
   <artifactId>gcs-connector</artifactId>
-  <version>hadoop3-2.2.10</version>
```
Comment (arunkumarchacko) [inline README.md:51]: I think https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/1080 covered this.

File: coverage/pom.xml [hunk 2]
```diff
@@ -21,11 +21,11 @@
   <parent>
     <groupId>com.google.cloud.bigdataoss</groupId>
     <artifactId>bigdataoss-parent</artifactId>
-    <version>3.0.0-SNAPSHOT</version>
```
Comment (arunkumarchacko) [inline coverage/pom.xml:24]: why is this not updated by the release automation?

Comment (singhravidutt) [reply coverage/pom.xml:24]: It did. Check commitId: [8d7c5cd](https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/1091/commits/8d7c5cd35f43067fe6ac4de7867e8a7dbb273a09)

Changes I have to rollback were specifically in commit: [13877a] https://github.com/GoogleCloudDataproc/hadoop-connectors/pull/1091/commits/b03bea4e0c36270f8d6cb60f10db1acf5013877a

As our automation script checks for version: `3.0.0` and replaced it with `3.0.1-SNAPSHOT`. It caused a wrong package update. This collision should not occur that frequently now.

**Review by arunkumarchacko — APPROVED** [review]

**Pull request merged**